src/main/resources/TotemCode/TotemMacros.xml

```diff
@@ -311,5 +311,27 @@
 &lt;input type="hidden" id="redirectURL" value="$doc.getURL('view','showEdit=true')"/&gt;
 &lt;input type="hidden" id="showIframe" value="1"/&gt;
 #end
+
+#**
+* Display the totem actions links : Edit totem, documentation ...
+*#
+#macro(displayTotemActionsLinks)
+#set($totemViewURL = $doc.getURL())
+#set($totemEditURL = $doc.getURL('view','showEdit=true'))
+#set($totemDocURL = $xwiki.getURL('TotemCode.Documentation','view'))
+{{html clean="false" wiki="true"}}
+&lt;div class="clearfix"&gt;
+&lt;div class="pull-right"&gt;
+ &lt;a class="btn btn-info" href="$totemDocURL" target="_blank"&gt;&lt;span class="glyphicon glyphicon-book"&gt;&lt;/span&gt; $services.localization.render('totem.edit.documentation')&lt;/a&gt;
```

Comment:
By using glyphicon, your application will not work with Colibri, but only with Flamingo. That could be an acceptable limitation. But in general, prefer:
```$services.icon.render('book')```

In colibri, you will have the silk icon, in Flamingo, you will have a font-awesome icon.

Here the list of available icons through this API: http://design.xwiki.org/xwiki/bin/view/Proposal/XWiki+Icon+Set